src/OneBlinkFormBase.tsx

```diff
@@ -53,6 +53,7 @@ export type BaseProps = {
   attachmentRetentionInDays?: number
   onSaveDraft?: (
     newDraftSubmission: submissionService.NewDraftSubmission,
+    backgroundUpload?: boolean,
```

Comment:
I know this is a bit of a hassle but can we please add this to the `NewDraftSubmission` in apps.